src/support/resolve-node-specifier.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright (c) 2019 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import resolveFrom from 'resolve-from';
+import {relativePath} from './path-utils';
+
+export default (modulePath: string, specifier: string): string => {
+  const resolved = resolveFrom.silent(modulePath, specifier);
```

Comment:
Does `require.resolve(specifier, {paths: [modulePath]})` work here? Then we wouldn't need the `resolve-from` dependency.

https://nodejs.org/api/modules.html#modules_require_resolve_request_options

Reply:
`require.resolve` didn't work for us in polymer-cli, but I forget exactly why. I would be curious the difference between `resolve` and `resolve-from` though.

Reply:
I tried this with some MWC components today and it didn't work with some relative imports that didn't have file extensions. `resolveFrom.silent()` appears to not resolve any relative file imports, but because it doesn't throw an exception, we don't notice it, and nice modules that use extensions everywhere still work.

So two things:
1. Use plain `resolveFrom`, catch the exception, and log it
2. Figure out why relative imports aren't resolved.

Reply:
Added several more tests for node specifier resolution https://github.com/Polymer/koa-node-resolve/pull/1/files#diff-3d314b7f8996847ddb370de05bc83913

Reply:
Also migrated away from `resolve-from` to `resolve` package and updated logic similar to https://github.com/Polymer/tools/blob/5b3b45abd815f4a70ad62ee5018369c12e7f7b7b/packages/analyzer/src/javascript/resolve-specifier-node.ts